hooks/hooks.go

```diff
@@ -26,11 +27,13 @@ const (
 	UpdateStatus          Kind = "update-status"
 	PreSeriesUpgrade      Kind = "pre-series-upgrade"
 	PostSeriesUpgrade     Kind = "post-series-upgrade"
+	SecretRotate Kind = "secret-rotate"
```

Comment:
not sure if it's github showing wrong alignment?

Reply:
yeah, i just fixed it :-)